homeassistant/components/aussie_broadband/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the Aussie Broadband integration."""
+
+DOMAIN = "aussie_broadband"
+
+CONF_USERNAME = "username"
+CONF_PASSWORD = "password"
```

Comment:
Use `CONF_USERNAME` and `CONF_PASSWORD` from https://github.com/home-assistant/core/blob/dev/homeassistant/const.py.